### ISSUE TITLE
Make a new release with relativity support

### ISSUE THREAD

**Issue by dfm** [exoplanet-dev/exoplanet#135]
Once https://github.com/dtamayo/reboundx/pull/51 is merged and released.

**Comment by dfm** [comment]
This is now released. @arjunsavel: v0.4.4 now has reboundx support. This possibly should have been the 0.5 release, but I've got some bigger changes that I want to make before then.

**Comment by ericagol** [comment]
Excellent!  Does that include jacobians?

**Comment by dfm** [comment]
@ericagol: it sure does!

**Comment by dfm** [comment]
ref: https://github.com/exoplanet-dev/exoplanet/pull/133

**Comment by ericagol** [comment]
@dfm @arjunsavel  Do you have a notebook you could loan me which shows how to use this?

I'm interested in simply computing derivatives and comparing with NbodyGradient - nothing fancy.

**Comment by dfm** [comment]
@ericagol: Can you open a new issue describing what you want and I'm sure that one of us can put something together.
